x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformIndexer.java

```diff
@@ -1155,25 +1155,31 @@ private SearchRequest buildQueryToUpdateDestinationIndex() {
 
     /**
      * Indicates if an audit message should be written when onFinish is called for the given checkpoint
-     * We audit the first checkpoint, and then every 10 checkpoints until completedCheckpoint == 99
-     * Then we audit every 100, until completedCheckpoint == 999
+     * We audit every checkpoint for the first 10 checkpoints until completedCheckpoint = 9.
```

Comment:
Please change to "until completedCheckpoint == 9" for consistency with below statements.